### DISCUSSION TITLE
How to create project repository

### DISCUSSION THREAD

**Discussion by zzulueta** [microsoft/AI-Chat-App-Hack#86] [Q&A] [answered]
Sorry, I am not an expert on github. I want to submit my project but I dont know how to get this Project Repository URL. How do I put my codespaces code to a project repository? :)

**Comment by zedhaque** [comment]
It's the URL of your GitHub repository. (e.g., where you see the codespace button) if AI-Chat-App-Hack was your project then the URL would be:

https://github.com/microsoft/AI-Chat-App-Hack

**Reply by zzulueta** [reply]
where can i find it here?
![image](https://github.com/microsoft/AI-Chat-App-Hack/assets/41460714/26179a7e-4bf4-4c33-83c4-1c4e9586d0a2)

**Comment by zedhaque** [comment, marked as answer]
Looks like this is your repo url
https://github.com/zzulueta/azure-search-openai-demo

Check the data folder - does the house1, house2 data is what you uploaded for your chat? Looks familiar?

**Reply by zzulueta** [reply]
oh cool. thanks!
